src/codegen/templates/property_deleter_template.py.mako

```diff
@@ -0,0 +1,37 @@
+<%def name="script_property_deleter(attribute)">\
+<%
+        from codegen.utilities.text_wrappers import wrap
+    %>\
+    @${attribute.name}.deleter
+    def ${attribute.name}(self):
+<%
+        argtypes = []
+        for handle_parameter in attribute.handle_parameters:
+            if handle_parameter.ctypes_data_type == 'ctypes.c_char_p':
+                argtypes.append('ctypes_byte_str')
+            else:
+                argtypes.append(handle_parameter.ctypes_data_type)
+    %>\
+    %if len(attribute.c_function_name) < 33:
```

Comment:
A comment here about why we care about length would be nice. It looks like we line-wrap if its too long? Alternatively, just always line-wrap to simply this code?